codegen/src/main/java/io/sundr/codegen/processor/JavaGeneratingProcessor.java

```diff
@@ -65,10 +65,10 @@ public void generateFromResources(TypeDef model, String resourceName) throws IOE
    */
   public void generateFromResources(TypeDef model, JavaFileObject fileObject, String resourceName) throws IOException {
     if (classExists(model)) {
-      System.err.println("Skipping: " + model.getFullyQualifiedName() + ". Class already exists.");
+      printSkipping("Class", model.getFullyQualifiedName());
       return;
     }
-    System.err.println("Generating: " + model.getFullyQualifiedName());
+    // System.err.println(String.format("Generating: %-100s\r", model.getFullyQualifiedName()));
```

Comment:
Is this comment intended?

Reply:
Yes, is intended, it can be removed as this make the output unnecessarily verbose.

But it can be uncommented too if you think this is needed.

Reply:
Let's remove it completely!

Reply:
Done!